### ISSUE TITLE
TODAY placeholder not localized

### ISSUE THREAD

**Issue by rakete1** [sinaatalay/RenderCV#111]
The placeholder TODAY in "last_updated_date_style" is always replaced by `Date.today().strftime("%B %Y")`. As it is not possible to call the `setlocale` function (see https://docs.python.org/3/library/locale.html) this funtions always returns an English string like "**June 2024**".
However it would be great if I could control the locale that is used via the yaml file so that this language matches my `locale_catalog` in the yaml.
Currently in the header it says for me now: "Zuletzt aktualisiert: **June** 2024" whereas the rest of the document and the dates are translated to German language by using my own `locale_catalog`.

**Comment by sinaatalay** [comment]
Thank you for finding this! It will be available in the next release.

**Comment by rakete1** [comment]
Yea. Sure. Happy you plan to integrate it!
Maybe there is a different or more elegant way as to call the `setlocale` function. I'm not sure. But I'm confident you will find a good way. I love your framework. Thank you for the great work!

**Comment by sinaatalay** [comment]
I like the full control over the PDF via the YAML file, so I ended up adding a new field to the `locale_catalog` field: [`full_names_of_months`](https://docs.rendercv.com/user_guide/structure_of_the_yaml_input_file/#locale_catalog-field). From now on, it will pull `TODAY`'s month name from there.

If you would like to test it, you can install RenderCV from the latest commit with the command below:

```
pip install "rendercv@git+https://github.com/sinaatalay/rendercv"
```

**Comment by rakete1** [comment]
Yea. Thats really consistent with the other options. I like it! Test was successful. Thanks again!

**Comment by sinaatalay** [comment]
This issue has been fixed with RenderCV v1.12.
